README.md

```diff
@@ -13,26 +13,29 @@ In order to set your environment up to run the code here, first install all requ
 pip install -r requirements.txt
 ```
 
+Rename example.env to .env and edit the variables appropriately.
+MODEL_TYPE supports LlamaCpp or GPT4All
```

Comment:
It'd be useful to explain in the readme what each variable in .env is used for and the possible values.

Reply:
Done. Updated to be easier to read too.